### DISCUSSION TITLE
Getting started docs with PNPM

### DISCUSSION THREAD

**Discussion by ericfennis** [nuxt/nuxt#22121] [Questions] [answered]
I noticed, on the Getting Started page of the documentation, if you look on the PNPM tab this code example is shown:
```sh
pnpm dlx nuxi@latest init <project-name>
```

But is it not easier to show it like this, or what is the difference?
```sh
pnpx nuxi@latest init <project-name>
```

**Comment by Jannchie** [comment, marked as answer]
It looks like they are the same.
https://stackoverflow.com/a/73428539/8625892

But `pnpx` seems to be a deprecated command.
https://github.com/pnpm/pnpm.io/blob/37f40f101b724c367c35a91d18e7dee8a2333d8b/versioned_docs_archived/version-6.x/pnpx-cli.md?plain=1#L8
https://github.com/orgs/pnpm/discussions/5590

**Reply by ericfennis** [reply]
Ahh, that's it, thanks for the answer!
